compiler/src/main/kotlin/dev/zacsweers/metro/compiler/fir/generators/ContributedInterfaceSupertypeGenerator.kt

```diff
@@ -138,6 +144,13 @@ internal class ContributedInterfaceSupertypeGenerator(
     return graphAnnotation.scopeArgument() != null
   }
 
+  override fun FirDeclarationPredicateRegistrar.registerPredicates() {
+    // Currently we only need to resolve qualifiers for dagger-anvil's 'rank' interop
```

Comment:
All the predicates should be registered if they are used for lookups. The other two predicates are only working because another extension registers a similar predicate using the same annotations. Registration is cheap as well, so every extension should really register the extensions it uses, to avoid breakage due to another extension changing what it registers.

Reply:
Thanks! I was curious how that was working since they weren't already being registered here